CLAUDE.md

```
@@ -11,10 +11,62 @@
 - Run feature tests: `./vendor/bin/pest tests/Feature`
 - Run unit tests: `./vendor/bin/pest tests/Unit`
+- Run browser tests: `php artisan dusk`
 
 ## Linting & Code Quality
 - Fix code style: `./vendor/bin/duster fix`
 - Run Pint: `./vendor/bin/pint`
+- Generate IDE helper files: `php artisan ide-helper:generate`
+
+## Gmail Integration
+
+### Authentication Setup
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)